plugins/main/public/controllers/overview/components/last-alerts-stat/last-alerts-stat.tsx

```diff
@@ -155,7 +156,12 @@ export function LastAlertsStat({
                 }}
                 href={discoverLocation}
               >
-                {statValue}
+                {
```

Comment:
I think this is no longer necessary because it is done in the formatUINumber function.